model/FeatureFlag/UniqueNumericQtiIdentifierQtiCreator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\FeatureFlag;
+
+use oat\tao\model\featureFlag\FeatureFlagCheckerInterface;
+use oat\tao\model\featureFlag\FeatureFlagConfigHandlerInterface;
+
+class UniqueNumericQtiIdentifierQtiCreator implements FeatureFlagConfigHandlerInterface
```

Comment:
I think this feature will be used by a customer using tao advance, no? 

Why not use the existing handler in this case ? https://github.com/oat-sa/extension-tao-deliver-connect/blob/develop/model/FeatureFlag/FeatureFlagQtiCreatorConfigHandler.php

Reply:
I wanted to keep all changes in taoQtiItems as it contain related business logic inside.

Reply:
Can we at least have a single handler to deal with all configs? IMO it is really not needed to create a handler per each config we change, specially cause you are targeting the QTI creator

Reply:
I have following handlers defined in cached container:
```
$instance->addClientConfigHandler('oat\\taoQtiItem\\model\\FeatureFlag\\FeatureFlagFlaHandler');
$instance->addClientConfigHandler('oat\\taoQtiItem\\model\\FeatureFlag\\UniqueNumericQtiIdentifierClientConfig');
$instance->addExtensionConfigHandler('taoQtiItem', 'qtiCreator', 'oat\\taoQtiItem\\model\\FeatureFlag\\UniqueNumericQtiIdentifierQtiCreator');
$instance->addClientConfigHandler('oat\\taoDeliverConnect\\model\\FeatureFlag\\FeatureFlagClientConfigHandler');
$instance->addExtensionConfigHandler('taoQtiItem', 'qtiCreator', 'oat\\taoDeliverConnect\\model\\FeatureFlag\\FeatureFlagQtiCreatorConfigHandler');
```
Do you want me to merge them into two handlers?

Reply:
🤦 Oh, they are dealing with different config files. Did not notice that. We need to keep 2 handlers indeed in this case. Forget what I said